programmers/난이도별/level01.나머지가_1이_되는_수_찾기/JiaLee0707.js

```diff
@@ -0,0 +1,7 @@
+function solution(n) {
+    var answer = n - 1;
+    for(let i=2; i<n; i++) {
+        if (n % i === 1) return i
+    }
+    return answer;
```

Comment:
이미 for문에서 ealry return 을 해주고 있기 때문에 answer 변수를 만들 필요가 없을 것 같아요!
만약 for문에서 return을 하지 못한다면 `n-1`값을 return 하도록 해줍시다 :)
```suggestion
    for(let i=2; i<n; i++) {
        if (n % i === 1) return i
    }
    return n - 1; // n-1 값 바로 return!
```

Reply:
불필요한 코드나 변수를 줄일 수 있도록 노력해야겠네요!
리뷰 감사합니다 😄